examples/kind/README.md

```diff
@@ -0,0 +1,51 @@
+# Use apiserver-network-proxy with KIND
+
+
+Change to the `examples/kind` folder and create a `kind` cluster with the `kind.config` file
+
+```sh
+$ kind create cluster --config kind.config
+Creating cluster "kind" ...
+DEBUG: docker/images.go:58] Image: kindest/node:v1.27.3@sha256:3966ac761ae0136263ffdb6cfd4db23ef8a83cba8a463690e98317add2c9ba72 present locally
+ ✓ Ensuring node image (kindest/node:v1.27.3) 🖼
+⠎⠁ Preparing nodes 📦 📦 📦
+
+This node has joined the cluster:
+* Certificate signing request was sent to apiserver and a response was received.
+* The Kubelet was informed of the new secure connection details.
+
+Run 'kubectl get nodes' on the control-plane to see this node join the cluster.
+ ✓ Joining worker nodes 🚜
+Set kubectl context to "kind-kind"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-kind
+
+Have a nice day! 👋
+```
+
+Once the cluster is ready install the `apiserver-network-proxy` components:
+
+```sh
+$ kubectl apply -f konnectivity-server.yaml
+clusterrolebinding.rbac.authorization.k8s.io/system:konnectivity-server created
+daemonset.apps/konnectivity-server created
+
+$ kubectl apply -f konnectivity-agent-ds.yaml
+serviceaccount/konnectivity-agent created
+```
+
+To validate that it works, run a custom image and try to get the logs:
+```sh
+$ kubectl run test --image httpd:2
+pod/test created
+$ kubectl get pods
+NAME   READY   STATUS              RESTARTS   AGE
+test   0/1     ContainerCreating   0          4s
+$ kubectl get pods
+NAME   READY   STATUS    RESTARTS   AGE
+test   1/1     Running   0          6s
+$ kubectl exec -it test bash
```

Comment:
nit: the comment above (and the e2e.yaml) both say `kubectl logs`, want to keep exec or make it consistent?

Reply:
I think we can merge and iterate , I just needed to proof that ipv6 works, I will circle back to this but can't say when, in the meantime this adds a net value

Reply:
oh, damn, I see now what you mean, let me change it

Reply:
fixed